### ISSUE TITLE
Pressing Esc after selecting an item quits the app.

### ISSUE THREAD

**Issue by ferdiebergado** [excalibur1234/pacui#37]
Is it possible to accomplish the behavior that when pressing escape after selecting an item should go back to the main menu?

**Comment by excalibur1234** [comment]
you are right that this behavior was lost with the recent addition of "bash strict mode", which quits the script whenever an error is encountered (and pressing ESC creates an error).

i try to remember to deactivate bash strict mode next time i do a release.

if you told me about this bug a couple of days earlier, i could have fixed it for the 1.10 release.

p.s.: if you want to deactivate strict mode in pacui yourself, simply comment the following lines out (this means you have to add a # symbol to the front of the line):
```
set -e                                                                              # exit script immediately, if any command exits with non-zero error code
set -u                                                                              # only allow previously defined variables
set -o pipefail                                                                     # if one command in a pipe fails, all fail (this is not default behavior!)
```

**Comment by excalibur1234** [comment]
[version 1.10.1](https://github.com/excalibur1234/pacui/releases/tag/1.10.1) disabled bash strict mode.

this should solve your problem for good.
